config.json

```diff
@@ -133,7 +133,12 @@
       "core": true,
       "difficulty": 3,
       "slug": "secret-handshake",
-      "topics": null,
+      "topics": [
+        "loops",
+	"transforming",
+        "conditionals",
+	"cryptography"
```

Comment:
Great suggestions! I would add `enumerations` since it uses an `enum` and maybe also `mathematics` and `bitwise_operations`? What do you think?

Also, the indentation seems to have gone a bit wrong. They should all be indented two spaces in from `topics`, like `loops` and `conditionals` are :)

Reply:
Oh and also `integers` would be a good topic :)

Reply:
Thanks for all the informations and suggestions. I'm up for adding more topics - I wasn't sure if there's a suggested/maximum number of topics for each exercise, so I kept it minimal.

Reply:
I don't think we have a suggested maximum or minimum number of topics at the moment. Adding topics is still a relatively new things so that might change in the future, but for the moment I think we can just add as many topics as we feel are relevant :)